src/common/proto/BUILD

```diff
@@ -14,19 +14,22 @@
 # limitations under the License.
 #
 #-------------------------------------------------------------------------------
+package(
+    default_visibility = ["//src:__subpackages__"],
+    features = ["layering_check"],
+    licenses = ["notice"],
+)
 
-package(default_visibility = ["//src:__subpackages__"])
-
-licenses(["notice"])
-
-# These files contain dummy marker libraries for indicating a dependency upon
-# protobuf libraries. We have to use them because Bazel cannot be configured
-# to just force explicit inclusion of protobuf dependencies. This causes
-# problems when we try to import Raksha into Google's internal repository.
-cc_library(
+# These files contain marker libraries for indicating a dependency upon
+# protobuf libraries. We have to use them because the external protobuf
+# library provides a single BUILD target that encompasses all protobuf
+# related headers. However, the internal targets are much more fine grained.
```

Comment:
Should probably be "Google-internal" rather than just "internal", as we expect that our readers in an open source repository may have their own default idea of "internal".

Reply:
Done.